data/bangs.json

```diff
@@ -109391,5 +109391,381 @@
     "t": "fqm",
     "u": "https://frequentmiler.com/?s={{{s}}}",
     "c": "Shopping"
-  }
+  },
+  {
+    "s": "Casal Sport",
+    "d": "www.casalsport.com",
+    "t": "casalsport",
+    "u": "https://www.casalsport.com/SearchDisplay?storeId=13151&sType=SimpleSearch&searchTerm={{{s}}}",
```

Comment:
Is the `storeId` parameter going to be a problem? Just tested and it is necessary, just worried it's going to always pull up results from the same store (same goes for other bangs here with the same param).

Reply:
As far as I understand the store ID seems to be more related to the actual 'web store' and not the actual physical stores.

Reply:
As far as I could find they are using IBM WebSphere Commerce and there storeid refers to a ibm wsc server itself. Not a physical store location.
![ResizedImage_2024-10-29_00-28-30_8863](https://github.com/user-attachments/assets/2b133fd1-b3fd-4abd-affd-bb147734e2b2)